src/lib/validateFrontmatter.ts

```diff
@@ -68,15 +72,37 @@ export async function validateFrontmatter(
       });
 
       pagesToReturn = updatedFiles;
+      status = 'autofixed';
+      this.log('✅ The following files have been automatically fixed:');
+      pagesToReturn.forEach((file, index) => {
+        const currentValidationResult = validationResults[index];
+        if (currentValidationResult.fixableErrorCount) {
+          this.log(
+            `- ${file.filePath} (${currentValidationResult.fixableErrorCount} issue${currentValidationResult.fixableErrorCount > 1 ? 's' : ''})`,
+          );
+        }
+      });
+
+      this.log();
+      this.log(
+        `Please review the changes. Once everything looks good, re-run \`${chalk.blue(`${this.config.bin} ${this.id}`)}\` to upload these files to ReadMe.`,
+      );
     }
 
     // also inform the user if there are files with issues that can't be fixed
     if (filesWithUnfixableIssues.length) {
-      this.warn(
-        `${filesWithUnfixableIssues.length} file(s) have issues that cannot be fixed automatically. The upload will proceed but we recommend addressing these issues. Please get in touch with us at support@readme.io if you need a hand.`,
-      );
+      if (status !== 'autofixed') {
+        this.warn(
+          `${filesWithUnfixableIssues.length} file(s) have issues that cannot be fixed automatically. The upload will proceed but we recommend addressing these issues. Please get in touch with us at support@readme.io if you need a hand.`,
+        );
+      } else {
+        status = 'autofixed-with-issues';
+        this.warn(
+          `${filesWithUnfixableIssues.length} file(s) have issues that cannot be fixed automatically. Autofixable issues have been addressed but we also recommend addressing these issues as well. Please get in touch with us at support@readme.io if you need a hand.`,
```

Comment:
```suggestion
          `${filesWithUnfixableIssues.length} file(s) have issues that cannot be fixed automatically. Autofixable issues have been corrected but we also recommend addressing these issues as well. Please get in touch with us at support@readme.io if you need a hand.`,
```
"addressed" and "fixed" feel kinda redundant, but I don't feel super strongly here

Reply:
works for me!